.github/workflows/test-cli.yml

```diff
@@ -35,13 +35,13 @@ jobs:
           python -m pip install --upgrade pip
           pip --version
           # Windows needs special treatment of cryptography for unknown reasons.
-          # LMDB also has special requirements that only apply to windows (patch-ng).
```

Comment:
Feels arbitrary but sure

Reply:
That was just to get it under 80 chars. 😂